cpp/oneapi/dal/algo/linear_kernel/backend/gpu/compute_kernel_dense_dpc.cpp

```diff
@@ -14,20 +14,79 @@
 * limitations under the License.
 *******************************************************************************/
 
+#define DAAL_SYCL_INTERFACE
```

Comment:
nip: I think this `#define` can be moved into "oneapi/dal/backend/interop/common_dpc.hpp" file